src/lib/config/ModelConfig.cc

```diff
@@ -886,6 +886,19 @@ void ModelConfig::postValidation() {
   // Record any unlinked port names
   for (const auto& prt : portnames)
     invalid_ << "\t- " << prt << " has no associated reservation station\n";
+
+  // Ensure LSQ-L1-Interface Load/Store Bandwidth is large enough for vector
+  // unit
+  if (configTree_["Core"]["Vector-Length"].as<uint16_t>() / 8 >
+      configTree_["LSQ-L1-Interface"]["Load-Bandwidth"].as<uint16_t>()) {
+    invalid_ << "\t- Vector-Length (bits) must be greater than Load-Bandwidth "
+                "(bytes)\n";
+  }
+  if (configTree_["Core"]["Vector-Length"].as<uint16_t>() / 8 >
+      configTree_["LSQ-L1-Interface"]["Store-Bandwidth"].as<uint16_t>()) {
+    invalid_ << "\t- Vector-Length (bits) must be greater than Store-Bandwidth "
+                "(bytes)\n";
+  }
```

Comment:
Similar checks should also be made for Streaming vector length, as this can a) be larger than the (sve) vector length, ad b) increase the size of Sve vector loads/stores when streaming mode is enabled.

Reply:
Good idea, I've added these checks in the latest commit.